cli/tests/integrations/common.py

```diff
@@ -630,7 +630,7 @@ def ssh_output(cmd):
     proc, master = popen_tty(cmd)
 
     # wait for the ssh connection
-    time.sleep(3)
+    time.sleep(5)
```

Comment:
I was going to file an issue to fix this "the right way", because it could be somewhat involved. But I can look into it now if you want.